pandas/tests/plotting/test_backend.py

```diff
@@ -36,3 +42,44 @@ def test_backend_is_correct(monkeypatch):
         pandas.set_option("plotting.backend", "matplotlib")
     except ImportError:
         pass
+
+
+@td.skip_if_no_mpl
+def test_register_entrypoint():
```

Comment:
this is failing for me locally

Reply:
What are you seeing?

Reply:
Oh, I'm guessing it's a `KeyError: 'pandas_plotting_backends'`?

You'll need to re-run `python -m pip install -e .` in your pandas directory. This adds the entrypoint.

Reply:
Yep that does fix it, thanks